topology/connection_creator.cpp

```diff
@@ -25,11 +25,10 @@
 namespace nest
 {
 
-const DictionaryDatum ConnectionCreator::dummy_param_ = new Dictionary;
-
 ConnectionCreator::ConnectionCreator( DictionaryDatum dict )
   : allow_autapses_( true )
   , allow_multapses_( true )
+  , allow_oversized_( false )
```

Comment:
what is this used for?

Reply:
I think I just added it because I noticed it wasn't initialized. I can remove it again.

Reply:
@jakobj If `allow_oversized_` is not initialized here, the test `test_oversized_mask.sli` fails. I don't understand why the test suddenly fails here and not in master though.

Reply:
uninitialized is always bad, thanks for fixing this.